scss/_settings_spacing.scss

```diff
@@ -21,6 +23,8 @@ $font-sizes: (
 ) !default;
 
 $line-heights: (
+  display: 11 * $sp-unit,
+  display-mobile: 9 * $sp-unit,
```

Comment:
```suggestion
  display-mobile: 8 * $sp-unit,
```

Reply:
Doesn't it make line-height too small? Lower part of "p" sticks out from the bottom of the box:

<img width="507" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/1b49b8dd-e214-4ffe-8940-30b3b264d718">